planck-cljs/src/planck/io.cljs

```diff
@@ -1,5 +1,5 @@
 (ns planck.io
-  (:require planck.core)
+  (:require [planck.core])
```

Comment:
If emacs wants it that way, I guess it can only be correct. 

Erik. 
-- 
i farta

> Den 28. jan. 2016 kl. 16.55 skrev Mike Fikes <notifications@github.com>:
> 
> In planck-cljs/src/planck/io.cljs:
> 
> > @@ -1,5 +1,5 @@
> >  (ns planck.io
> > -  (:require planck.core)
> > +  (:require [planck.core])
> Hah, looks like your IDE insists on changing this line.
> 
> —
> Reply to this email directly or view it on GitHub.
>